mk/xamarin.mk

```diff
@@ -1,5 +1,5 @@
 ifdef ENABLE_XAMARIN
-NEEDED_MACCORE_VERSION := 62a239f3f89573dd68c120993cd1dfeb166f8c9c
+NEEDED_MACCORE_VERSION := 12ecfd8249cab8d3867a779e5ce633f97bb3825b
```

Comment:
This hash is from maccore's `d16-2` branch, it should be the tip of maccore's `xcode11` branch.